tests/contrib/openai/test_openai_llmobs.py

```diff
@@ -141,6 +234,21 @@ def test_completion_stream(self, openai, ddtrace_global_config, mock_llmobs_writ
             ),
         )
 
+    def test_chat_completion_proxy(self, openai, ddtrace_global_config, mock_llmobs_writer, mock_tracer):
+        """Ensure llmobs records are not emitted for chat completion endpoints when the base_url is specified."""
+        with patch.object(openai._base_client.SyncAPIClient, "post") as mock_completions_post:
+            mock_completions_post.return_value = MOCK_OPENAI_CHAT_COMPLETIONS_RESPONSE
+            model = "gpt-3.5-turbo"
+            input_messages = [
+                {"role": "system", "content": "You are a helpful assistant."},
+                {"role": "user", "content": "Who won the world series in 2020?"},
+                {"role": "assistant", "content": "The Los Angeles Dodgers won the World Series in 2020."},
+                {"role": "user", "content": "Where was it played?"},
```

Comment:
i'm noticing a lot of these in `test_openai_llmobs.py` and `test_openai_v1.py` - can we extract this out as a utils constant?